Makefile

```diff
@@ -1,21 +1,11 @@
 # simple makefile to simplify repetitive build env management tasks under posix
 
-# caution: testing won't work on windows, see README
-
 PYTHON ?= python
-CYTHON ?= cython
-PYTEST ?= pytest
-
-# skip doctests on 32bit python
-BITS := $(shell python -c 'import struct; print(8 * struct.calcsize("P"))')
-
-all: clean inplace test
```

Comment:
isn't `all` the default? I kinda do like to have _a_ default here though.

Reply:
Yes `all` with what `make` (without any arguments) does. What would you want it to be: `dev-meson` + `pytest` only `dev-meson`.

I am still hoping that in the not so far future we switch to `spin` so, I am not super motivated to add new `make` patterns that people will get used to.

Reply:
yeah fair.

TBH, since meson, since it automatically rebuilds on import, I haven't used `make` at all. So the only thing I'd really like to have is a shortcut to the `pip` command. Like `make install` or something.

Reply:
something that doesn't necessarily mention `meson` in the command and is what comes to mind.